frontend/src/bundles/common/components/input/styles.module.scss

```diff
@@ -0,0 +1,115 @@
+@import url("../../../../assets/css/styles.scss");
```

Comment:
I think we don't need this import because we already import styles in common file

Reply:
Do you mean index.tsx from the frontend root? Okay, I agree. Should I remove it right now or when I will come back to it next time when Josh or Vova fix all variables with propper names?

Reply:
Maybe it's better to do now but I'm not sure

Reply:
It is already imported here - frontend/src/index.tsx